source/loop.cpp

```diff
@@ -42,6 +42,24 @@ Loop::Loop(char *config)
   parameters.use_flux_limiting = string2bool(get_element_text(root,"use_flux_limiting"));
   parameters.calculate_dem = string2bool(get_element_text(root,"calculate_dem"));
   parameters.use_adaptive_solver = string2bool(get_element_text(root,"use_adaptive_solver"));
+  parameters.radiation = get_element_text(root,"radiation");
+  if (parameters.radiation == "power_law")
+  {
+      parameters.use_variable_abundances = false;
+  }
+  else if( (parameters.radiation == "variable") ||
+          (parameters.radiation == "photospheric") ||
+          (parameters.radiation == "coronal") )
+  {
+      parameters.use_variable_abundances = true;
```

Comment:
It is confusing that this is set to true even when the radiation uses only photospheric or only coronal abundances.

Reply:
Instead of `use_variable_abundances`, why not make it `use_power_law_losses` and then invert the logic?

Reply:
How about renaming to `use_lookup_table_losses` or something?  I think that more accurately describes the usage of this `bool`.